crates/bevy_window/src/window.rs

```diff
@@ -19,6 +20,24 @@ use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
 
 use crate::VideoMode;
 
+/// Default string used for the window title.
+///
+/// It will try to use the name of the current exe if possible, otherwise it defaults to "App"
+static DEFAULT_WINDOW_TITLE: LazyLock<String> = LazyLock::new(|| {
+    #[cfg(feature = "std")]
+    {
+        std::env::current_exe()
+            .ok()
+            .and_then(|current_exe| Some(format!("{}", current_exe.file_stem()?.to_string_lossy())))
+            .unwrap_or_else(|| "App".to_owned())
+    }
+
```

Comment:
You could also use the `file` macro to get the source filename. That might be better in both cases and then you wouldn't need special handling:

https://doc.rust-lang.org/core/macro.file.html

Reply:
It will just give the name of window.rs file, not the name of the example. 

Also, even if it was the name of the top level file, this won't work for users with a cargo project that uses main.rs.